modules/rosa-sts-interactive-mode-reference.adoc

```diff
@@ -40,29 +40,29 @@ You can create a {product-title} cluster with the AWS Security Token Service (ST
+|Enable compute node autoscaling. The autoscaler adjusts the size of the cluster to meet your deployment demands. The default is `No`.
 
 |`Compute nodes`
 |Specify the number of compute nodes to provision into each availability zone. Clusters deployed in a single availability zone require at least 2 nodes. Clusters deployed in multiple zones must have at least 3 nodes. The default is `2`.
 
 |`Machine CIDR`
-|Specify the machine IP address range. A minimum IP address range of 128 addresses, using the subnet prefix `/25`, is supported for single availability zone deployments. A minimum address range of 256 addresses, using the subnet prefix `/24`, is supported for deployments that use multiple availability zones. The default is `10.0.0.0/16`.
+|Specify the IP address range for machines (cluster nodes), which must encompass all CIDR address ranges for your VPC subnets. Subnets must be contiguous. A minimum IP address range of 128 addresses, using the subnet prefix `/25`, is supported for single availability zone deployments. A minimum address range of 256 addresses, using the subnet prefix `/24`, is supported for deployments that use multiple availability zones. The default is `10.0.0.0/16`. This range must not conflict with any connected networks.
 
 |`Service CIDR`
-|Specify the IP address range for services. The OpenShift SDN allows only one address block for services. The address block must not overlap with any other address block. The default is `172.30.0.0/16`.
+|Specify the IP address range for services. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`. It is recommended that they are the same between clusters.
 
 |`Pod CIDR`
-|Specify the pod IP address range. The OpenShift SDN network plug-in supports multiple cluster networks. The address blocks for multiple cluster networks must not overlap. The ranges must be large enough to accommodate your workload. The default is `10.128.0.0/14`.
+|Specify the IP address range for pods. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`. It is recommended that they are the same between clusters.
 
 |`Host prefix`
-|Specify the subnet prefix length to assign to each individual node. For example, if the host prefix is set to `23`, each node is assigned a `/23` subnet in the CIDR address range. The default is `23`.
+|Specify the subnet prefix length assigned to pods scheduled to individual machines. The host prefix determines the pod IP address pool for each machine. For example, if the host prefix is set to `/23`, each machine is assigned a `/23` subnet from the pod CIDR address range. The default is `/23`, allowing 512 cluster nodes and 512 pods per node, both of which are beyond our maximum supported maximums.
```

Comment:
I noticed we said "maximum" twice in the last three words.  Can we do two things here:
1. remove the first "maximum". So that it reads "...both of which are beyond our supported maximums".
2. Can we link "supported maximums" to https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#tested-cluster-maximums_rosa-limits-scalability?

Reply:
@0kashi - I updated the text, removing the first "maximum" and I added the link to the additional resources since we cannot link directly from a module to a module.

Do these changes work for you? Thanks!

Reply:
Thanks for the update.  Since we cannot link from module to module, can we at least add to the end something like "See additional resources below" in parentheses?  So that a customer can know there is a link in the addition resources.